setup.py

```diff
@@ -34,6 +34,8 @@
     author = 'Paul Barrett, Phil Hodge',
     author_email = 'help@stsci.edu',
     description = 'Tools for STIS (Space Telescope Imaging Spectrograph)',
+    long_description='README.md',
```

Comment:
Should this be the contents of the `README.md` file rather than a string with the name of it?

https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata

```
Set the value of long_description to the contents (not the path) of the README file itself.
```

Reply:
Fair enough - maybe we don't need the whole README.md on the PyPI page - for example jwst just has a single line of plain text.  How about
long_discription='Tools for analyzing STIS data from the Hubble Space Telescope'
long_description_content_type='text/plain'

Reply:
Works for me.